### PR TITLE
fix: Add known acronyms for type generator

### PR DESCRIPTION
**Pull request by jasonvigil**
These were used in WorkstationConfig.

### CODE REVIEW

**Review by jingyih — COMMENTED** [review]
/lgtm

File: dev/tools/controllerbuilder/pkg/codegen/typegenerator.go [hunk 1]
```diff
@@ -348,6 +348,20 @@ func isAcronym(s string) bool {
...
+	case "fs":
+		return true
+	case "pd":
+		return true
+	case "kms":
+		return true
+	case "gce":
+		return true
+	case "vtpm":
+		return true
```
Comment (yuwenma) [inline dev/tools/controllerbuilder/pkg/codegen/typegenerator.go:364]: Cool, can I ask where you get this list?

File: dev/tools/controllerbuilder/pkg/codegen/typegenerator.go [hunk 2]
```diff
@@ -348,6 +348,20 @@ func isAcronym(s string) bool {
...
+	case "fs":
+		return true
+	case "pd":
+		return true
+	case "kms":
+		return true
+	case "gce":
+		return true
+	case "vtpm":
```
Comment (yuwenma) [inline dev/tools/controllerbuilder/pkg/codegen/typegenerator.go:363]: Shall this be "vTPM}"?

File: dev/tools/controllerbuilder/pkg/codegen/typegenerator.go (hunk 1 above)
Comment (jasonvigil) [reply dev/tools/controllerbuilder/pkg/codegen/typegenerator.go:364]: WorkstationCluster

File: dev/tools/controllerbuilder/pkg/codegen/typegenerator.go (hunk 2 above)
Comment (jasonvigil) [reply dev/tools/controllerbuilder/pkg/codegen/typegenerator.go:363]: That would mean the variable ends up named `EnablevTPM`, which is not ideal. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/603f8c85cdd15ab4c605eb1034d22d4d06b4fa14/apis/workstations/v1alpha1/config_types.go#L145

Comment (yuwenma) [reply dev/tools/controllerbuilder/pkg/codegen/typegenerator.go:363]: Fair enough.

**Pull request merged**